packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -555,25 +568,25 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     return channelId;
   }
 
-  /**
-   * Jumpstart a torrent wire download, after being stopped by a peer.
-   */
+  /** Jumpstart a torrent wire download, after being stopped by a peer. */
   protected jumpStart(torrent: ExtendedTorrent, wire: PaidStreamingWire) {
     if (torrent.done) {
       log.info('<< JUMPSTART: FINISHED');
-      log.trace({torrent, wire});
+      log.trace({torrent, wire: wire.paidStreamingExtension.serialize()});
       return;
     }
     log.info({requests: wire.requests}, `<< START ${wire.paidStreamingExtension.peerAccount}`);
     log.trace({pieces: torrent.pieces});
     (torrent as any)._updateWireWrapper(wire); // TODO: fix this type, if its the solution
```

Comment:
Do we know what this does or if it is necessary?

Reply:
I don't know what `log.trace` does, I just know that using the wire object directly has problems (it doesn't appear correctly if the wire is not serialized.

Reply:
Just to be clear, I am referring to `(torrent as any)._updateWireWrapper(wire);`

Reply:
I don't remember, with today's PR I will check and document it.